### PR TITLE
[ADD] medical_base_kanban: Abstract KanBan logic

### PR DESCRIPTION
**Pull request by lasley**
- Add module for abstract KanBan logic for Medical resources

### CODE REVIEW

File: medical_base_kanban/README.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+=====
+
+The ``medical.base.kanban`` model should be polymorphic inherited in order to
+add KanBan functionality to any record
+
+.. code-block:: python
+
+    class MyModel(models.Model):
+        _name = 'my.model'
+        _inherits = {'medical.base.kanban': 'kanban_id'}
```
Comment (lasley) [inline medical_base_kanban/README.rst:21]: This is now direct _inherit

File: medical_base_kanban/README.rst [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Medical Base -  Kanban
+======================
+
+This module provides abstract KanBan logic for use in Medical Resources.
+
+
+Usage
+=====
+
+The ``medical.base.kanban`` model should be polymorphic inherited in order to
```
Comment (lasley) [inline medical_base_kanban/README.rst:14]: No poly

File: medical_base_kanban/README.rst [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+    class MyModel(models.Model):
+        _name = 'my.model'
+        _inherits = {'medical.base.kanban': 'kanban_id'}
+        kanban_id = fields.Many2one(
+            string='KanBan'
+            comodel_name='medical.base.kanban',
+            required=True,
+            ondelete='cascade',
+            default=lambda s: s.env['medical.base.kanban']._default_kanban_id()
```
Comment (lasley) [inline medical_base_kanban/README.rst:27]: Update

**Comment by tedsalmon** [comment]
👍

**Comment by lasley** [comment]
Wait what? Why did you merge an in progress? I even had notes of things I needed to change...

**Pull request merged**